docs/tutorials/torch_bert_inference.md

```diff
@@ -117,7 +117,7 @@ torch_config = torch_blade.config.Config()
 torch_config.enable_mlir_amp = False # disable mix-precision
 with torch.no_grad(), torch_config:
   # BladeDISC torch_blade optimize will return an optimized TorchScript
-  optimized_ts = torch_blade.optimize.optimize(model, allow_tracing=True, model_inputs=tuple(inputs))
+  optimized_ts = torch_blade.optimize(model, allow_tracing=True, model_inputs=tuple(inputs))
```

Comment:
The associated documents, like torch_bert_inference.md, should be revised. Pls update